lib/CL/pocl_util.c

```diff
@@ -411,103 +294,7 @@ void pocl_command_enqueue (cl_command_queue command_queue,
 
 }
 
-char* pocl_get_process_name ()
-{
-  char tmpStr[64], cmdline[512], *processName = NULL;
-  FILE *statusFile;
-  int len, i, begin;
-
-  snprintf(tmpStr, 64, "/proc/%d/cmdline", getpid());
-  statusFile = fopen(tmpStr, "r");
-  if (statusFile == NULL)
-    return NULL;
-
-  if (fgets(cmdline, 511, statusFile) != NULL)
-    {
-      len = strlen(cmdline);
-      begin = 0;
-      for (i=len-1; i>=0; i--)     /* Extract program-name after last '/' */
-        {
-          if (cmdline[i] == '/')
-            {
-              begin = i + 1;
-              break;
-            }
-        }
-      processName = strdup(cmdline + begin);
-    }
-
-  fclose(statusFile);
-  return processName;
-}
-
-static int cache_lock_initialized = 0;
-static pocl_lock_t cache_lock = POCL_LOCK_INITIALIZER;
-
-void
-pocl_check_and_invalidate_cache (cl_program program,
-                  int device_i, const char* device_tmpdir)
-{
-  int cache_dirty = 0;
-  char *content = NULL, *s_ptr, *ss_ptr;
-  int read = 0;
-
-  POCL_LOCK(cache_lock);
 
-  if (!pocl_get_bool_option("POCL_KERNEL_CACHE", POCL_BUILD_KERNEL_CACHE))
-    {
-      cache_dirty = 1;
-      goto bottom;
-    }
-
-  /* If program contains "#include", disable caching
```

Comment:
Yup, removed all the old code for grepping #includes. Yeah i guess a test would be nice :)